README_LINUX.md

```diff
@@ -0,0 +1,131 @@
+
+
+### (Recommended) Verify you have a credential manager installed to store your credentials 
+
+Recent versions of Git install a credential manager to store your credentials so you don't have to put in the credentials for every request.  
+It is highly recommended you check that you have a [credential manager installed and configured](https://github.com/microsoft/Git-Credential-Manager-Core#linux)
+
+
+
+### Step 1. Clone the repositories
+
+Let's assume you are cloning the engine and the project to **/home/your_username/git** where **your_username** is your Linux username and **/home/your_username** is your home folder. The rest of the readme will reference this path as **~**.
+
+```shell
+> cd ~
+> mkdir git
+> cd git
+# Clone the engine.
+> git clone https://github.com/o3de/o3de
+Cloning into 'o3de'...
+
+# Clone the project into a folder outside your engine repository folder.
+> git clone https://github.com/o3de/o3de-multiplayersample.git
+Cloning into 'o3de-multiplayersample'...
+```
+
+
+### Step 2. Register the engine and project 
+
+```shell
+# Register the engine (only need to do this once).
+> cd ~/git/o3de
+> ./scripts/o3de.sh register --this-engine
+
+# Register the project.
+> ./scripts/o3de.sh register -p ~/git/o3de-multiplayersample
+```
+
+### Step 3. Configure and build with Project-centric approach 
+
+This option will output all the project binaries in the project's build folder e.g. **~/git/o3de-multiplayersample/build**.
+
+```shell
+> cd ~/git/o3de-multiplayersample
+> mkdir build
+> cd build
+
+# Configure.
+> cmake .. -G "Ninja Multi-Config" -DCMAKE_BUILD_TYPE=profile -DLY_3RDPARTY_PATH=~/ws/3rdParty -DCMAKE_C_COMPILER=clang-12 -DCMAKE_CXX_COMPILER=clang++-12 -DLY_PROJECTS="~/git/o3de-multiplayersample"
+
+# Build the Editor, game launcher and server launcher.
+> cmake --build . --config profile --target Editor
+> cmake --build . --config profile --target MultiplayerSample.GameLauncher
+> cmake --build . --config profile --target MultiplayerSample.ServerLauncher
+```
+
+#### Step 3b. (Optional) Build and Run Multiplayer Unittests
+
+```shell
+> cd ~/git/o3de-multiplayersample/build
+
+> cmake --build . --config profile --target Multiplayer.Tests
+# Run unittests and benchmarks
+> ./bin/profile/AzTestRunner ./bin/profile/libMultiplayer.Tests.so AzRunBenchmarks
+> ./bin/profile/AzTestRunner ./bin/profile/libMultiplayer.Tests.so AzRunUnitTests
+```
+
+### Step 4. Setup Client and Server
+
+Under engine root, create 2 files: **client.cfg** and **server.cfg**. File ~/git/o3de/client.cfg should contain:
+
+```shell
+connect
+```
+
+File ~/git/o3de/server.cfg should contain:
+
+```shell
+host
+LoadLevel Levels/SampleBase/SampleBase.spawnable
+```
+
+### Step 5. Verify Asset Processor 
+
+```shell
+> cd ~/git/o3de-multiplayersample/build
+> cmake --build . --config profile --target AssetProcessor
+# launch Asset Processor and verify MultiplayerSample assets are good
```

Comment:
capital “Launch” to match other comments starting uppercase

Reply:
Corrected.